test/extended/builds/pipeline_jenkins_e2e.go

```diff
@@ -24,7 +22,7 @@ import (
 	"github.com/openshift/origin/test/extended/util/jenkins"
 )
 
-var _ = g.Describe("[Slow]jenkins repos e2e openshift using slow samples pipeline build", func() {
+var _ = g.Describe("[Slow]jenkins repos e2e openshift using slow openshift pipeline build", func() {
```

Comment:
Don't we have a `[Feature:Jenkins]` tag?

Reply:
No we just use "openshift pipeline" as the ginkgo focus in `openshift-tests` .... I vaguely remember way back when there was an issue with multiple `Feature` directives (and the pipeline_origin.go has `[Feature:Builds]` ) but I can experiment with such a switch as well once I narrow down which tests are reliable.